examples/StandardFirmataBLE/bleConfig.h

```diff
@@ -106,6 +106,15 @@ BLEStream stream;
 // BLEStream stream(BLE_REQ, BLE_RDY, BLE_RST);
 // #endif
 
+/*
+ * Adafruit Bluefruit nRF52 Boards
+ */
+#ifdef ARDUINO_NRF52_ADAFRUIT
+#include "utility/BluefruitStream.h"
+BLEStream stream;
+#endif
```

Comment:
Can you open a separate PR with just the Boards.h changes. I'll then leave this PR open until the analogRead issues are resolved.